src/app/community/community.component.ts

```diff
@@ -57,7 +57,8 @@ export class CommunityComponent implements OnInit, OnDestroy {
   resizeCalendar: any = false;
   deviceType: DeviceType;
   deviceTypes = DeviceType;
-  isLoading: boolean;
+  isLoading = true;
```

Comment:
Let's have these variables be `isLoadingInit` and `isLoadingMore`. Gives a little more clarity on the two to have both be verbose.